src/index.html

```diff
@@ -22,6 +22,428 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="image"
+        href="#"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__item">
```